.eslintignore

```diff
@@ -1 +1 @@
-!.eslintrc.js
+.eslintrc.js
```

Comment:
any reason this was changed? we'd like ESLint to ensure that the config file matches style as well.

Reply:
It was getting a "Parsing error: 'parserOptions.project' has been set to @typescript-eslint-parser'. The file does not match your project config: .eslintrc.js.
The file must be included in at least one of the projects provided."
If you have an idea to make it work, I have not.